src/util/mock/mock.ts

```diff
@@ -0,0 +1,14 @@
+import {Page} from '@playwright/test';
+import {Mockable} from "../../testcase";
+import {Youtube} from "./youtube";
+
+export class Mock implements Mockable{
```

Comment:
It's not clear to me that loading a Mock here actually does something with YouTube. I'm not clear why we need this extra Mock class. Would it be clearer to have `YouTubeMock Implements Mockable`?

Reply:
This is definitely bad naming. This extra Mock class was using in some projects to call all the mocks mock method. As we only have youtube here for now, makes not a lot of sense. Maybe renaming to EveryIframeServiceMock would make sense.